applications/FluidDynamicsApplication/custom_elements/two_fluid_navier_stokes.cpp

```diff
@@ -157,7 +164,54 @@ void TwoFluidNavierStokes<TElementData>::CalculateLocalSystem(
                     ComputeGaussPointEnrichmentContributions(data, Vtot, Htot, Kee_tot, rhs_ee_tot);
                 }
 
-                CondenseEnrichment(data, rLeftHandSideMatrix, rRightHandSideVector, Htot, Vtot, Kee_tot, rhs_ee_tot);
+                if (rCurrentProcessInfo[SURFACE_TENSION]){
+                    // Surface tension coefficient is set in material properties
+                    const double surface_tension_coefficient = this->GetProperties().GetValue(SURFACE_TENSION_COEFFICIENT);
+                    Matrix int_shape_function, int_shape_function_enr_neg, int_shape_function_enr_pos;
+                    GeometryType::ShapeFunctionsGradientsType int_shape_derivatives;
+                    Kratos::Vector int_gauss_pts_weights;
```

Comment:
```suggestion
                    Vector int_gauss_pts_weights;
```
I think you don't need the namespace specifier.

Reply:
done for all instances.